### PR TITLE
consider minus when reading data

### PR DESCRIPTION
**Pull request by dawonahn**

### CODE REVIEW

File: src/GTA.cpp [hunk 1]
```diff
@@ -173,9 +173,16 @@ void Read_Tensor(char* Path) {
...
 				flag = 1;
 			}
+			else if (tmp[j] == '-') {
+				flag2 = 1;
+			}
 		}
 		if(flag==0) val = idx;
-		X.value[i] = val;
+        if(flag2 == 0){ 
+		    X.vale[i] = val;
```
Comment (sejoonoh) [inline src/GTA.cpp]: X.value[i]=val;

File: src/GTA.cpp [hunk 2]
```diff
@@ -173,9 +173,16 @@ void Read_Tensor(char* Path) {
...
+        }else{
+            X.value[i] = 0 * val;
```
Comment (sejoonoh) [inline src/GTA.cpp]: X.value[i]=-val;

**Review by sejoonoh — CHANGES_REQUESTED** [review]
Please review the comments

**Comment by dawonahn** [comment]
> Please review the comments

Sorry for mistakes, I edited them again.

**Review by sejoonoh — APPROVED** [review]

**Pull request merged**